### PR TITLE
Update DZ60 Information

### PR DESCRIPTION
**Pull request by mrsendyyk**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mrsendyyk** [comment]
Cc: @fauxpark @drashna @tzarc @noroadsleft :)

**Comment by drashna** [comment]
@mrsendyyk just a heads up, you don't need to tag us in each of the PRs.  We will get to them sooner or later, as time and life permits. 

That said, I do like your enthusiasm. :)

File: keyboards/readme.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ These keyboards are part of the QMK repository, but their manufacturers are not
 * [Atreus](/keyboards/atreus) &mdash; Made by Technomancy.
 * [Bantam44](/keyboards/bantam44) &mdash; It is a 44-key 40% staggered keyboard.
 * [DataHand](/keyboards/handwired/datahand) &mdash; DataHand keyboard converted to use a Teensy board.
+* [DZ60 60% Mechanical Keyboard PCB](/keyboards/dz60) &mdash; A Customizable 60% Mechanical Keyboard PCB.
```
Comment (fauxpark) [inline keyboards/readme.md:41]:
```suggestion
```

File: keyboards/readme.md [hunk 2]
```diff
@@ -46,7 +47,7 @@ These keyboards are part of the QMK repository, but their manufacturers are not
 * [JD45](/keyboards/jd45) &mdash; Another Geekhack community project, designed by jdcarpe.
 * [JJ40](/keyboards/jj40) &mdash; ps2avrGB based ortholinear sold by KPrepublic
 * [JJ50](/keyboards/jj50) &mdash; A compact 50% (5x12) ortholinear keyboard sold by KPrepublic.
-* [KBD75](/keyboards/kbd75) &mdash; A 75% keyboard made by made by KBDFans.
+* [KBD75](/keyboards/kbdfans/kbd75) &mdash; A 75% keyboard made by made by KBDFans.
```
Comment (fauxpark) [inline keyboards/readme.md:50]:
```suggestion
* [KBD75](/keyboards/kbd75) &mdash; A 75% keyboard made by made by KBDFans.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Looks good other than the changes to `keyboards/readme.md`, which I think we are probably going to remove altogether; it's way out of date and just too much of a hassle to maintain.

File: keyboards/readme.md (hunk 1 above)
Comment (mrsendyyk) [reply keyboards/readme.md:41]: Ok, done.

File: keyboards/readme.md (hunk 2 above)
Comment (mrsendyyk) [reply keyboards/readme.md:49]: Ok, done.

**Comment by mrsendyyk** [comment]
Sorry @drashna, i accidentally dismissed your stale review.

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**